WcaOnRails/app/assets/stylesheets/users.scss

```diff
@@ -19,3 +19,16 @@ input[type="radio"].center-vertically {
   vertical-align: middle;
   height: 100%;
 }
+
+.input-group.date.datetimepicker {
+  width: 100%;
+}
+
+.form-group {
```

Comment:
I still don't see the problem you're trying to describe.

If you clicked outside of the input, how else would you expect to get to
the datepicker again, if not by clicking?

I tried both allowInputToggle and focusOnShow, but got errors saying the
option is unrecognized.

I could move the focus back to the input if you enter an invalid date. Is
that what you mean?

On Tue, Feb 9, 2016 at 12:29 AM, Jeremy Fleischman <notifications@github.com
> wrote:

> In WcaOnRails/app/assets/stylesheets/users.scss
> <https://github.com/cubing/worldcubeassociation.org/pull/379#discussion_r52260849>
> :
>
> > @@ -19,3 +19,16 @@ input[type="radio"].center-vertically {
> >    vertical-align: middle;
> >    height: 100%;
> >  }
> > +
> > +.input-group.date.datetimepicker {
> > +  width: 100%;
> > +}
> > +
> > +.form-group {
>
> We moved away from the button because we both thought that was the only
> way to get the calendar to pop up when you focus an <input>. I'm pretty
> sure one of those options I linked you to will allow us to do both, however.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/379/files#r52260849>
> .
>